flow/FastRef.h

```diff
@@ -156,6 +156,7 @@ class Reference
 
 	bool isValid() const { return ptr != NULL; }
 	explicit operator bool() const { return ptr != NULL; }
+	P*& changePtrUnsafe() { return ptr; }
```

Comment:
Do we still need this now that we're punting on re-implementing IReplicationPolicy in flat buffers?

Reply:
```suggestion
```